arch/lws/eventloop_lws_http.c

```diff
@@ -0,0 +1,611 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ *
+ *    Copyright 2023 (c) Fraunhofer IOSB (Author: Noel Graf)
+ */
+
+#include <open62541/plugin/eventloop.h>
```

Comment:
LWS is not an architecture.
It currently depends on POSIX.
Hence I would move it to /arch/posix/lws.